manifest.json

```diff
@@ -567,8 +567,8 @@
       "required": true
     },
     {
-      "projectID": 284229,
-      "fileID": 2968355,
+      "projectID": 737252,
```

Comment:
Following a mishap where a mod disappeared from the manifest during merges (see 1.2.2.1's changelog), we have taken to a convention of keeping project nodes sorted by projectID. Please remove the original node and create a new one in the appropriate position in the manifest.